src/test/java/com/wikia/webdriver/elements/oasis/pages/InfoboxBuilderPage.java

```diff
@@ -261,7 +357,21 @@ public InfoboxBuilderPage setLongLabelNameAndVerifyBreakLine(
     return this;
   }
 
-  public InfoboxBuilderPage scrollAndSelectLastComponent() {
+  public InfoboxBuilderPage setAndVerifyHeaderNameWithIndex(int index, String labelName) {
```

Comment:
I would rename it to just "setAndVerifyHeaderName", parameter already indicated it's by index.